package.json

```diff
@@ -93,7 +93,7 @@
     "@material/typography": "^1.0.0",
     "@types/chai": "^4.1.7",
     "@types/classnames": "^2.2.6",
-    "@types/enzyme": "^3.1.15",
+    "@types/enzyme": "3.1.15",
```

Comment:
@moog16 it turns out that this is the last type compatible version for `@types/react ~16.7.22`